signalfx/client.py

```diff
@@ -49,6 +56,25 @@ def send_event(self, event_type, dimensions=None, properties=None):
         logging.debug('Sending event to SignalFx: %s', data)
         return data
 
+    @staticmethod
+    def get_aws_unique_id(timeout=DEFAULT_TIMEOUT):
```

Comment:
Done... I added a new constant called DEFAULT_AWS_TIMEOUT  (to avoid confusion with the ingest workflow, which is also HTTP)